notebooks/takeout_442_metadata.txt

```diff
@@ -1,106 +0,0 @@
-original_sentence_id	sentence	locale
```

Comment:
Isn't this file needed to run the colab? Also, I was not actually possible for me to download just the txt file. Just copy-pasting the contents worked, so it's not a real issue, but maybe it could be added to the zip, so only only thing would need to be downloaded?

Reply:
I think because we are collaborators on each other's repositories and that's why we can't download so I guess this is not a big issue.